extension/lib/shopgate/read_model/Shopgate.js

```diff
@@ -44,3 +44,8 @@
  * @property {string} storeHash
  * @property {string} clientSecret
  */
+
+/**
+ * @typedef {Object} DeleteProductsInput
+ * @property {[string]} CartItemIds
```

Comment:
The way we document arrays is for example `string[]`.

Reply:
corrected